### DISCUSSION TITLE
Overriding YAML?

### DISCUSSION THREAD

**Discussion by natereedcb** [nielsbasjes/yauaa#898] [Q&A] [answered]
Would it be possible to override the the location of the YAML files when constructing the UserAgentAnalyzer?

I'd like to be able to get updates to these files without redeploying my service. It looks like there isn't currently a way to do this.  My thinking was they could be fetched from github and if there was a way to specify a url for the files, I'd just need to create a hook of some kind to force the creation of new UAA instances.

Thoughts?

**Comment by nielsbasjes** [comment, marked as answer]
In the builder you can drop the prepackaged config and then load your own. 
Do note that sometimes I have to change the java code to make it all work.
So I do not guarantee this will work.

**Reply by natereedcb** [reply]
Thanks! I found the methods in the builder:

`dropDefaultResources`
`addResources`

There is a nice test for this, too, in TestResourceLoading.java.
